### ISSUE TITLE
Vue3 

### ISSUE THREAD

**Issue by AtefR** [uwla/vue-data-table#24]
I'm trying to use this package with vue 3 on my project and I get this error
`Uncaught (in promise) TypeError: Cannot read property '_c' of undefined`

Code:

```javascript
<template>
    <app-layout>
        <template #header>
            <h2 class="font-semibold text-xl text-gray-800 leading-tight">
                Clients
            </h2>
        </template>

        <div class="py-12">
            <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">
                <div class="bg-white overflow-hidden shadow-xl sm:rounded-lg">
                    <div class="m-10 ">
                        <data-table v-bind="bindings"/>
                    </div>
                </div>
            </div>
        </div>
    </app-layout>
</template>

<script>
import AppLayout from '@/Layouts/AppLayout'
import DataTable from '@andresouzaabreu/vue-data-table';
import "@andresouzaabreu/vue-data-table/dist/DataTable.css";

export default {
    components: {
        AppLayout,
        DataTable
    },
    props: ['clients'],
    computed: {
        bindings() {
            return {
                columns: [
                    {
                        key: "name",
                    },
                    {
                        key: "email",
                        title: "Email Address",
                        sortable: false,
                    },
                ],
                data: this.clients
            }
        }
    }
}
</script>
```

**Comment by AndreSouzaAbreu** [comment]
I havent' tried VueDataTable with Vue3 (only with Vue 2). I'll take a look at it (may take some days as I'm busy new). A quick question that may help debugging: is the `this.client` an array of objects?And each object of the array has the attributes `email` and `name` ?

**Comment by AtefR** [comment]
Yes this.clients is array of objects.
also I've tried with the data array in your reply, unfortunately I got the same error.

**Comment by AndreSouzaAbreu** [comment]
> Yes this.clients is array of objects.
> also I've tried with the data array in your reply, unfortunately I got the same error.

It should have worked. I will see if I can reproduce the error using Vue3 on the next couple of days (may take some days bc now I don't have much free time) and then release a patch version of VueDataTable fixing it

**Comment by shahidali81** [comment]
@AndreSouzaAbreu any update upon it because it is not working with Vue3.

**Comment by AndreSouzaAbreu** [comment]
@shahidali81 I'm test it out with Vue3 next week or so. 

If you want to, you can help me speed this up by creating some demo repo which
I can clone and test it locally.

I haven't yet because I haven't even tried Vue3

**Comment by AndreSouzaAbreu** [comment]
I tested it today and I can confirm the behavior. Probably Vue3 has introduced some breaking changes that
make my plugin not work. I'll have to check out what these changes are

**Comment by AndreSouzaAbreu** [comment]
I gotta say I have no idea of what to do...
Since I don't know where to start, this can take me some time (maybe a long time) to fix this bug

**Comment by AndreSouzaAbreu** [comment]
This issue hasn't been fixed even after 9 months :rofl:

I wanna fix it because I do wanna use my own plugin with Vue 3, but I'm still
lost to figure out which breaking changes introduced by Vue 3 made my plugin
not work.

Help is welcome

**Comment by uwla** [comment]
Vue3 support coming soon!

The Vue3 branch is passing all tests and build.

I'll just add a demo app to make sure it works not just locally.

**Comment by uwla** [comment]
Vue3 support is now available at versions `2.0.0` or above!
